src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1518,59 +1518,88 @@ The view of Workflows from Catalog in Studio and from Catalog Portal follow the
+
+==== Removing Objects inside bucket
+Select the objects to remove (CTRL + left click to select multiple objects) and click on the `trash` button. 
+
+WARNING: All the bucket content will be removed from the database and it cannot be restored.
+
+==== Exporting/Importing Objects into the Catalog
+Catalog Objects can be exported to a local archive, and share them easily by email, cloud, etc.  The same archive can be imported again in the Catalog on the same scheduler or in a different one.
+
+TIP: Use this feature to easily share Catalog objects across different environments.
+
+To export the selected objects as a ZIP archive, press the download button from portal view. 
+
+image::download-catalog-objects.png[align=left]
+
+To import, either a downloaded catalog archive zip file or a single object, click on the upload button and follow the instructions :
+
+image::upload-catalog-objects.png[align=left]
+
+* *Object Kind*: The kind of object to add. New Kinds can be added by clicking the `plus` button.
+
+* *Content Type*: The Content Type is used to indicate the media type of the resource. New content Types can be added by clicking the `plus` button. 
+
+WARNING: In case of uploading an archive, all the objects in the zip must have the same Kind and Content Type.
 
-==== Removing Workflows inside bucket
-For removing Workflows, first the user should select the bucket containing the Workflows. Then the user can select multiple Workflows with pressed CTRL button. To delete the selected Workflows push the `trash` button.
 
-===== Importing/Exporting Workflows into the Catalog
-You can export Workflows to an archive, to share them more easily (email, usb key,..), and then import the entire archive from the Catalog.
+==== Catalog Object Versioning
+The HEAD of a Catalog object corresponds to the last commit and it is displayed on the portal. 
 
-To export the selected Workflows as a ZIP archive from Catalog portal: select a bucket, then the targeted Workflows (you can select multiple Workflows with pressed CTRL button). Press "Download Workflows" button from portal view. You'll get a ZIP file that can be transferred on an other Scheduler.
-To import the zip archive of Workflows to the Catalog from portal, select the bucket where the Workflows will be added. Then press `Upload Workflows` button in portal and follow the instructions `Upload a ZIP archive of Workflows` opened window.
+In order to see all the commits of the selected Object, please click on the `clock` button inside the description area.
 
-==== List revisions of Workflow
-The last revision of the Catalog is always displayed on the portal. In order to see all revisions of selected Workflow, please click on `clock` button inside description area.
-The revisions are shown in chronological order by commit time.
+The commits are shown in chronological order by commit time. 
```

Comment:
in 'the' chronological

Reply:
I think "in chronological order" is correct